python/plugins/processing/algs/otb/maintenance/README

```diff
@@ -9,17 +9,17 @@ Set OTB environment
 --------------------
```

Comment:
Sidenote:
if you rename this file to README.md QGIS will render it with styling